test/Vault.spec.ts

```diff
@@ -647,92 +651,98 @@ describe("Vault", () => {
   });
 
   describe("sponsor", () => {
+    it("reverts if msg.sender is not sponsor", async () => {
```

Comment:
I guess we are missing a test where we add the role to a different account and it sponsors?

Reply:
the other tests check for account owner. do you mean test if for another account? like bob or something

Reply:
yap, give the role to bob and see if he can sponsor

Reply:
should be solved. check latest